docusaurus.config.ts

```diff
@@ -71,6 +71,7 @@ const config: Config = {
     prism: {
       theme: prismThemes.github,
       darkTheme: prismThemes.dracula,
+      additionalLanguages: ['rust'],
```

Comment:
@yurikoinaba can we fix the syntax highlighting issue ?

Reply:
I think it is working for Rust, 
<img width="986" alt="Screen Shot 2024-05-14 at 1 47 33 PM" src="https://github.com/HorizenLabs/zkverify-docs/assets/20520369/45baa9ec-bc7d-4a7c-867a-74f93ae38ebe">

Do you want it also added for Solidity?

Reply:
Solidity isn't currently supported, so we would have to custom add it.
https://github.com/FormidableLabs/prism-react-renderer/blob/master/packages/generate-prism-languages/index.ts#L9-L23

Reply:
@yurikoinaba It seams not working (at least on my side). I'm trying it by `yarn start` and I cannot see the syntax highlighting on my browser page, I also tried to add some code snipp in js but alse these simple exmples are not hightlighet.

Did you tryed it on your side?

Reply:
> Solidity isn't currently supported, so we would have to custom add it. https://github.com/FormidableLabs/prism-react-renderer/blob/master/packages/generate-prism-languages/index.ts#L9-L23

ok nvm at the moment, thank you very much. closing this

Reply:
@yurikoinaba, you are right, `rust` is already supported out of the box by prism in the default list that you pointed. 

But `solidity` is supported yet https://github.com/PrismJS/prism/blob/master/components/prism-solidity.js and should be enough add it into the `aditionalLanguages` array in `prism` session.